types/ApiResponses.ts

```diff
@@ -319,6 +321,10 @@ export interface GetUserActivityStreamResponse extends ApiResponse {
   activity: PublicActivity[];
 }
 
+export interface GetVisibleResumesResponse extends ApiResponse {
+  resumes: ResumeModel[];
+}
```

Comment:
nit: this would need to be `PublicResume[]` after we change our API to return public resumes